modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderReducer.ts

```diff
@@ -679,6 +679,15 @@ export function ObjectFolderReducer(state: TState, action: TAction): TState {
 			};
 		}
 
+		case TYPES.SET_HANDLE_CONNECTION: {
```

Comment:
should we rename this `SET_HANDLE_CONNECTION` to `SET_NODE_HANDLE_CONNECTION `as well?